.github/workflows/main.yml

```diff
@@ -110,6 +110,15 @@ jobs:
               TARGET_TRIPLE: wasm32-wasip1-threads
               THREAD_MODEL: posix
 
+          - name: Test SIMD
+            os: ubuntu-24.04
+            clang_version: 16
+            test: true
+            upload: wasm32-wasi
```

Comment:
You might want to change this upload ID to avoid the name conflict causing the CI error.

Reply:
Sure, sounds fine to me.